server/services/discord/init.js

```diff
@@ -31,121 +31,124 @@ client.once('ready', () => {
-          },
-          {
-            name: 'sendMessageUser',
-            description: 'Send a message to a user',
-            isEnable: true,
-            Parameters: {
-              create: [
-                {
-                  name: 'messageContent',
-                  description: 'The content of the message',
-                  isRequired: true
-                },
-                {
-                  name: 'UserId',
-                  description: 'The user id where the message will be send',
-                  isRequired: true
-                }
-              ]
+  try {
+    const discord = await database.prisma.Service.create({
+      data: {
+        name: 'Discord',
+        description: 'Discord service',
+        isEnable: true,
+        Actions: {
+          create: [
+            {
+              name: 'onMessage',
+              code: 'D-01',
+              description: 'When a message is sent',
+              isEnable: true,
+              Parameters: {
+                create: [
+                  {
+                    name: 'channelId',
+                    description: 'The channel id where the message is sent',
+                    isRequired: true
+                  }
+                ]
+              }
+            },
+            {
+              name: 'onVoiceChannel',
+              code: 'D-02',
+              description: 'When a user join a voice channel',
+              isEnable: true,
+              Parameters: {
+                create: [
+                  {
+                    name: 'channelId',
+                    description: 'The channel id where the user join',
+                    isRequired: true
+                  }
+                ]
+              }
+            },
+            {
+              name: 'onReactionAdd',
+              code: 'D-03',
+              description: 'When a user add a reaction to a message',
+              isEnable: true
+            },
+            {
+              name: 'onMemberJoining',
+              code: 'D-04',
+              description: 'When a user join a guild',
+              isEnable: true
             }
-          },
-          {
-            name: 'changeActivity',
-            description: 'Change the activity of the User',
-            isEnable: true,
-            Parameters: {
-              create: [
-                {
-                  name: 'activity',
-                  description: 'The activity of the user',
-                  isRequired: true
-                }
-              ]
+          ]
+        },
+        Reactions: {
+          create: [
+            {
+              name: 'sendMessageChannel',
+              code: 'D-01',
+              description: 'Send a message on a channel',
+              isEnable: true,
+              Parameters: {
+                create: [
+                  {
+                    name: 'messageContent',
+                    description: 'The content of the message',
+                    displayName: 'string'
```

Comment:
There is not anymore `displayName` member.
But there is `isRequired` now

Reply:
It was forgotten at the merge